<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppCard</class>
 <widget class="QWidget" name="AppCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="h_box">
     <item>
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start_button">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop_button">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="min_button">
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="max_button">
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="action_update">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="action_remove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="action_rename">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="action_edit_path">
   <property name="text">
    <string>Edit Path</string>
   </property>
  </action>
  <action name="action_auto_update">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Update</string>
   </property>
  </action>
  <action name="action_detach_on_close">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Detach On Close</string>
   </property>
  </action>
  <action name="action_auto_start">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Start</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
